app/src/main/java/com/example/roomremix2/ui/login/LoginActivity.kt

```diff
@@ -115,7 +115,6 @@ private lateinit var binding: ActivityLoginBinding
         Toast.makeText(applicationContext, errorString, Toast.LENGTH_SHORT).show()
     }
 }
-
```

Comment:
Formatting can, and should, be performed automatically, and consistently, by the IDE.  Formatting code is really programmer's preference anyway, and doesn't necessarily decrease technical debt.